recipe/meta.yaml

```diff
@@ -9,20 +9,19 @@ source:
   sha256: d77dcfa183fb74fa93dce91f3b762cc9dc316c58598112c508a2ea2d873cfcf0
 
 build:
-  number: 0
+  noarch: python
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
 
   run:
-    - python
-    - enum34  # [py<34]
+    - python >=3.6
     - future
     - requests
-    - typing  # [py<=37]
```

Comment:
The selector was wrong, as typing is available in all the Python versions we're using.

Reply:
<img width="691" alt="image" src="https://user-images.githubusercontent.com/2394761/234024422-9e0c78c6-da6c-4bf9-97f5-f00f63267f66.png">
https://anaconda.org/conda-forge/typing